docs/components/attention/react.md

```diff
@@ -23,55 +25,68 @@ import { Attention } from '@warp-ds/react';
 
 ```js
 function Example() {
-  const [show, setShow] = React.useState(false);
-  const targetEl = React.useRef();
+  const [show, setShow] = React.useState(false)
+  const targetEl = React.useRef()
 
   return (
     <div>
       <h1
         ref={targetEl}
         onMouseEnter={() => setShow(true)}
         onMouseLeave={() => setShow(false)}
+        onFocus={() => setShow(true)}
+        onBlur={() => setShow(false)}
+        onKeyDown={handleKeyDown}
+        tabIndex={0}
```

Comment:
I think we could make an example that looks like the React one ([code](https://github.com/warp-ds/react/blob/next/packages/attention/stories/Attention.stories.tsx#L133-L160)). The target does look like a button there, so probably there's no need for `h1` wrapper.
<img width="231" alt="Screenshot 2023-11-03 at 15 40 26" src="https://github.com/warp-ds/tech-docs/assets/41303231/251cb124-b466-48de-834a-53bce32a1d82">

Reply:
Ok! So should I remove the `<h1>` and move the ref to the `<Button>`-tag? :)

Reply:
I think so. Since it's a tooltip that usually opens on hover/focus, you could move the onFocus/onBlur to the Button,  too. Wdyt?

Reply:
Ah, I see you did that already in [c767176](https://github.com/warp-ds/tech-docs/pull/127/commits/c7671769c11e65e8fb7a79d2138533223c0f72a1), so then yes, the `ref` should also be set on the Button 😊